extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -257,8 +258,9 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
      * @param systemProperties System level properties (Part A) that a user can add to the telemetry item
      */
     public sendPageViewInternal(pageView: IPageViewTelemetryInternal, properties?: { [key: string]: any }, systemProperties?: { [key: string]: any }) {
-        if (typeof document !== "undefined") {
-            pageView.refUri = pageView.refUri === undefined ? document.referrer : pageView.refUri;
+        let _document = getDocument();
+        if (_document) {
```

Comment:
Will this need an explicit undefined check still? Or does using the utility function prevent crashes for where document doesn't exist?

Reply:
The utility takes care of the undefined check and explicitly returns null if document is undefined or already null